### ISSUE TITLE
My overall impression

### ISSUE THREAD

**Issue by alvarovillena** [mikeudacity/robo-pace-betatest#103]
I think this comic is a great idea to introduce some basic content of the course. It is fun, and makes you feel part of the team. It reminds me to those games or books where you have to do something in order to continue the storyline.

The challenges are easy if you have some background, but some parts can be hard if you never worked with them. In my opinion, 2D rotation matrices should be explained before E10, because it's easy to understand them with a short explanation.

The main issue of this lesson is E12. The code given is not working, and every answer is flagged as correct.

Maybe you can add a challenge based on simple localization, like in the AI for robotics course.

**Comment by cangelogamboa** [comment]
Good idea! I will have to see what we can do to incorporate these ideas.

I believe I fixed the issue with E12 - or at least I made it less broken. You should now be able to run the code and get 'only' an error message.

**Comment by stupsi99** [comment]
Yes, agree, the comic is a cool way to do it. Seeing as this whole thing is designed to get people excited about the program, I would also consider a cheat sheet after the harder tasks (which explains the solution in a few words, not just a link to a website).
E18 and the checkboxes is a bit silly though, considering that most applicants are beyond high school?! I think it would be better to have a teaser about the program. Maybe a video of the first sample & return mission project!

**Comment by cangelogamboa** [comment]
Good point! Thanks for letting me know your thoughts on the last quiz.  I will keep these comments in mind as I make changes to the course.
